Utils/azuretools-core/src/com/microsoft/azuretools/sdkmanage/AzureManager.java

```diff
@@ -41,6 +42,8 @@
 
     AppPlatformManager getAzureSpringCloudClient(String sid);
 
+    MySQLManager getMySQLManager(String sid);
```

Comment:
make it `default`? otherwise may cause error for its other implementations

Reply:
let us refactor this case in future in one time ? just like as other resources such as spring clound, web app, etc.